### DISCUSSION TITLE
Bugs in Anura3D v2023 GiD Problemtype

### DISCUSSION THREAD

**Discussion by albuquerqueraniel** [Anura3D/Anura3D_OpenSource#124] [General]
Dear all,

I've found some bugs in the .GOM file generated with the Anura3D v2023 GiD Problemtype:

1. When you define an axisymmetric simulation you got the error: 
" severe (408): fort: (7): Attempt to use pointer CALPARAMS when it is not associated with a target". 
The reason is that the wrong flag is being write in the $$DIMENSION at the GOM file. It is writing "2D-**A**xi**s**symmetric", when it should be "2D-axisymmetric".

2. The contacts properties are being write wrong, causing the following error: 
"forrtl: severe (59): list-directed I/O syntax error, unit 21, (...)"
This is the way it's being write in the GOM file:
![1](https://github.com/Anura3D/Anura3D_OpenSource/assets/45863863/8df60944-0479-4b4c-951d-a3634bf9512e)

And this is how it should be:

![2](https://github.com/Anura3D/Anura3D_OpenSource/assets/45863863/703ae280-b0e2-49c8-a2b9-dde7f42daca2)

3. The reaction forces flag are also being write wrong. These are the reaction surfaces:

![3](https://github.com/Anura3D/Anura3D_OpenSource/assets/45863863/82a5c61c-c610-41c0-a2a2-9c28de254087)
 
But the parameters values of $$START_OUTPUT_REACTION_FORCES wrote in the GOM are equal to the values of $$START_BODY_CONTACT_2D:

![5](https://github.com/Anura3D/Anura3D_OpenSource/assets/45863863/adb9ea24-d7ce-4ff1-adc2-932eb0cbbf13)

When I delete the contact properties, in the GiD, and generate the new files, this is the $$START_OUTPUT_REACTION_FORCES flag:

![4](https://github.com/Anura3D/Anura3D_OpenSource/assets/45863863/c0316505-d405-4501-b598-c85678a4f460)

When it should be something like this:

![6](https://github.com/Anura3D/Anura3D_OpenSource/assets/45863863/edc23f64-e564-4ec5-a638-90754578f7e4)

Best regards,
Raniel

**Comment by FrancescaCeccato** [comment]
Thank you for your feedback! We'll look into it.

**Comment by gdicarluccio** [comment]
Hi Raniel,
Thanks for your feedback! I just uploaded some changes in the ProblemType to fix these issues.
Could you please check if your model works now?
